Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM python:3.8
```

Comment:
Just to keep the number of pace containers in check, would it work with 
`FROM ghcr.io/noaa-gfdl/pace:3.8`

That container has 

gcc 13.2.0, 
netcdf-c 4.9.2 (hdf5 1.14.2)
netcdf-fortran 4.6.1
mpich 4.1.2
git 2.42.0
miniconda 22.11.1

installed.  The container can be rebuilt with openmpi in place of mpich; and with different version of packages (as long as they are availableon spack)

Reply:
I think the default for mpi should be openmpi as that is what we have been using on the development box and gaea

Reply:
I'm not opposed to that, as long as it's broadly available. Can you point me to the dockerfile for the image and I'll try building using that as a base

Reply:
The dockerfile is [here](https://github.com/mlee03/pace/blob/pace-ci-testing/Dockerfile).  The already built image can be pulled with `docker pull ghcr.io/noaa-gfdl/pace:3.8`.  I'll push the image with openmpi installed next week to the registry.

Reply:
As many of the larger computer systems we work with are currently HPE/Cray systems with a vendor implementation of MPICH, it wouldn't be a bad idea to test both OpenMPI and MPICH. Perhaps we should discuss at the next NASA-NOAA tag-up.

Reply:
I'm having a bit of trouble building the docker image from ghcr.io/noaa-gfdl/pace:3.8 on my mac. I'd also like to keep this similar. to the pyFV3 container as well. Would it be alright to keep this container as-is?